File: WebAPI/API/Models/Quiz/QuizResponseDTO.cs
using System;
using System.Collections.Generic;
using System.Data.Entity.Core.Common.CommandTrees;
using System.Linq;
using System.Web;

namespace API.Models
{
    public class QuizResponseDTO
    {
        public int Id { get; set; }

        public string Author { get; set; }

        public string Title { get; set; }

        public bool IsPublic { get; set; }

        public string Description { get; set; }

        public string ShareCode { get; set; }
    }
}

File: Android/app/src/main/java/org/equipe4/quizplay/model/transfer/QuizResponseDTO.java
package org.equipe4.quizplay.model.transfer;

import java.io.Serializable;
import java.util.Date;

public class QuizResponseDTO implements Serializable {
    public int id;

    public String author;

    public String title;

    public boolean isPublic;

    public String description;

    public String shareCode;

    public int numberOfQuestions;

    public Date date;
}
